src/db.cpp

```diff
@@ -4852,33 +4857,6 @@ void reset_zone(int zone, int reboot)
           last_cmd = 0;
       }
       break;
-    case 'H':                 /* loads a Matrix file into a host */
```

Comment:
This will break existing zoneloads (e.g. the snowflake key, which is a file that loads into a specific host for download).

Reply:
Thanks for the extra TODO, I'll be adding this back in when I've finished other code.